mqt/bench/benchmarks/qiskit_application_nature/groundstate.py

```diff
@@ -16,18 +16,16 @@ def create_circuit(molecule, basis: str = "sto3g"):
     """
 
     try:
-        from qiskit_nature.drivers import Molecule
+        from qiskit_nature.converters.second_quantization import QubitConverter
```

Comment:
is it necessary to wrap this in a `try ... except` block anymore? 
MQT Bench has no optional dependencies anymore and everything is installed per default.
Please also check this for the other "application" benchmarks.

Reply:
Done for all application benchmarks.